src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Common.ResourceManager.ScenarioTests.Netcore.csproj

```diff
@@ -1,12 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <Import Project="..\..\..\..\tools\Common.Netcore.Dependencies.targets" />
-
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <AssemblyName>Microsoft.Azure.Commands.ScenarioTest.Common</AssemblyName>
     <RootNamespace>Microsoft.Azure.Commands.ScenarioTest</RootNamespace>
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <OutputType>Exe</OutputType>
+    <RuntimeIdentifiers>win10-x64;linux-x64;osx.10.12-x64</RuntimeIdentifiers>
```

Comment:
[RuntimeIdentifiers](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj#runtimeidentifiers) are only used for [publishing](https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-publish?tabs=netcore2x), and should not be required.

Reply:
Yes, I had forgotten about the experiment to package as an exe and run tests from the package.  Will remove these.